### DISCUSSION TITLE
Is per-pixel scrolling possible?

### DISCUSSION THREAD

**Discussion by diocletiann** [neovide/neovide#1567] [Ideas]
The jerky scrolling when using the Macbook trackpad is the only thing holding me back from using neovide/neovim full time, I suspect I'm not alone. This would be a killer feature and bridge the gap with other GUI editors.

**Comment by LamprosPitsillos** [comment]
have you tried `neovide --multigrid` ?

**Reply by diocletiann** [reply]
Yes I use multigrid, which is fine with a mouse wheel. Not so much with a trackpad when trying to smoothly drag the canvas like every other gui.

**Reply by LamprosPitsillos** [reply]
try with mouse support and without
`set mouse=` to disable
`set mouse=a` to enable

**Reply by diocletiann** [reply]
`set mouse=a` was enabled and results in the jerky, row-based movement I'm trying avoid. `set mouse=` disables the trackpad.

**Reply by LamprosPitsillos** [reply]
Last thing i can think of is this app
https://mos.caldis.me/
I never used Mac nor this app ( can't tell if it will work) but i think i saw a video sometime ago about smooth scrolling and mac and it mentioned this app.

**Reply by diocletiann** [reply]
I appreciate the help but I think we're discussing different things. The issue is not the OS, it's that Neovide scrolls per row instead of per pixel.
